hack/make-bundle-vars.mk

```diff
@@ -96,9 +96,15 @@ ODF_CONSOLE_IMG_TAG ?= latest
 ODF_CONSOLE_IMG_LOCATION ?= quay.io/ocs-dev
 ODF_CONSOLE_IMG ?= $(ODF_CONSOLE_IMG_LOCATION)/$(ODF_CONSOLE_IMG_NAME):$(ODF_CONSOLE_IMG_TAG)
 
+PROMETHEUS_BUNDLE_NAME ?= prometheus-operator
+PROMETHEUS_BUNDLE_IMG_NAME ?= $(PROMETHEUS_BUNDLE_NAME)-bundle
+PROMETHEUS_BUNDLE_IMG_TAG ?= v4.10.0
```

Comment:
if the idea is to have this deployed along w/ odf, this tag is a bit misleading, can we follow whatever exists for ocs operator?

Reply:
Have updated to odf-prometheus-operator